x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -415,17 +417,23 @@ static RoleDescriptor kibanaSystem(String name) {
                         TransportUpdateSettingsAction.TYPE.name()
                     )
                     .build(),
+                // For source indices of the Cloud Detection & Response (CDR) packages that ships a
+                // transform
                 RoleDescriptor.IndicesPrivileges.builder()
-                    .indices("logs-wiz.vulnerability-*")
+                    .indices("logs-wiz.vulnerability-*", "logs-wiz.cloud_configuration_finding-*")
                     .privileges("read", "view_index_metadata")
                     .build(),
+                // For alias indices of the Cloud Detection & Response (CDR) packages that ships a
+                // transform
                 RoleDescriptor.IndicesPrivileges.builder()
                     // manage privilege required by the index alias
-                    .indices("security_solution-*.vulnerability_latest")
+                    .indices("security_solution-*.vulnerability_latest", "security_solution-*.misconfiguration_latest")
```

Comment:
so far our naming convention was plural

vulnerabilit**ies**, finding**s**. Also alert**s** or log**s**.
Maybe worth to keep it this way as a convention

Reply:
The convention I see in other packages to have data sources named as singular, same for ti latest IoC transforms. So I'd leave the more common convention and maybe change our naming when we get to getting rid of `logs-*` prefix